easybuild/easyconfigs/a/Autoconf/Autoconf-2.69-GCCcore-6.4.0.eb

```diff
@@ -20,6 +20,8 @@ source_urls = [GNU_SOURCE]
 sources = [SOURCELOWER_TAR_GZ]
 checksums = ['954bd69b391edc12d6a4a51a2dd1476543da5c6bbf05a95b59dc0dd6fd4c2969']
 
+patches = ['config.guess_ppc64le.patch']
```

Comment:
@omula Please rename to something like `Autoconf-2.69_config.guess_ppc64le.patch`, and include a checksum for the patch itself (`eb --inject-checksums` can do this for you).